src/Symfony/Component/Cache/Tests/Adapter/CouchbaseCollectionAdapterTest.php

```diff
@@ -20,8 +20,7 @@
  * @requires extension couchbase <4.0.0
  * @requires extension couchbase >=3.0.0
  *
- * @group integration
- * @group legacy
+ * @group integration legacy
```

Comment:
can you please swap them, so that at least we can grep for 'group legacy' and find this one?